app/src/main/java/com/github/fi3te/notificationcron/data/local/AppDatabase.kt

```diff
@@ -16,12 +19,19 @@ abstract class AppDatabase : RoomDatabase() {
 
         private var instance: AppDatabase? = null
 
-        private val MIGRATION_1_2: Migration = object : Migration(1, 2) {
+        val MIGRATION_1_2: Migration = object : Migration(1, 2) {
             override fun migrate(database: SupportSQLiteDatabase) {
                 database.execSQL("ALTER TABLE notification_cron ADD COLUMN enabled INTEGER DEFAULT 1 NOT NULL")
             }
         }
 
+        val MIGRATION_2_3: Migration = object : Migration(2, 3) {
+            override fun migrate(database: SupportSQLiteDatabase) {
+                database.execSQL("ALTER TABLE notification_cron ADD COLUMN position INTEGER")
+                database.execSQL("UPDATE notification_cron SET position = id WHERE position IS NULL;")
```

Comment:
Please unify the use of semicolons in database.execSQL calls.

Reply:
Done!